diff -u a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -71,15 +71,15 @@
 	$(CC) $(CFLAGS) -c $< -o $@
 
 ap51-flash: $(OBJS) $(LIN_OBJS) Makefile
-	$(CC) $(CFLAGS) $(OBJS) $(LIN_OBJS) -lpcap -o $@
+	$(CC) $(CFLAGS) $(OBJS) $(LIN_OBJS) $(LDFLAGS) -lpcap -o $@
 	$(STRIP) $@
 
 ap51-flash-static: $(OBJS) $(LIN_OBJS) Makefile
-	$(CC) $(CFLAGS) $(OBJS) $(LIN_OBJS) -lpcap -static -o $@
+	$(CC) $(CFLAGS) $(OBJS) $(LIN_OBJS) $(LDFLAGS) -lpcap -static -o $@
 	$(STRIP) $@
 
 ap51-flash.exe: $(OBJS) $(WIN_OBJS) Makefile
-	$(CC) $(CFLAGS) -LWpdPack/Lib/ -DWIN32 -D_CONSOLE -D_MBCS $(OBJS) $(WIN_OBJS) -lwpcap -static -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) -LWpdPack/Lib/ -DWIN32 -D_CONSOLE -D_MBCS $(OBJS) $(WIN_OBJS) -lwpcap -static -o $@
 	$(STRIP) $@
 
 kernel.o: $(EMBED_KERNEL)
